backend/app/utilities/db_utils.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, sessionmaker
+
```

Comment:
Let's just add `load_dotenv()` here so that we don't have to rely on `main.py` to load the environment variables, otherwise we'll probably have this issue again.